.phpunit.result.cache

```diff
@@ -0,0 +1 @@
+{"version":1,"defects":[],"times":{"ArrayAccessItemsTest::testBasicArrayAccessItems":0.025,"ArrayAccessItemsTest::testArrayAccessItemsWithIndexChanged":0,"ArrayAccessItemsTest::testArrayAccessItemsWithIndexChangedByMethod":0,"ContractedItemsTest::testBasicContractedItems":0.001,"DecoratorTest::testBasicArrayAccessItems":0.001,"NowChangedTest::testChangingNow":0,"NowChangedTest::testChangingNowByMethod":0,"PeriodActionsTest::testChangingPeriodForMonth":0.001,"PeriodActionsTest::testChangingPeriodForShorterMonth":0.001,"PeriodActionsTest::testChangingPeriodForYear":0.007,"PeriodActionsTest::testChangingPeriodForLeapYear":0.007,"PeriodActionsTest::testChangingPeriodForNumberOfTiles":0.002,"PeriodAligningTest::testAligningPeriodForDailyHeatmap":0.001,"PeriodAligningTest::testAligningPeriodForDailyHeatmapFromBothSides":0.001,"PeriodAligningTest::testAligningPeriodForNumberOfTiles":0,"PeriodChangedTest::testChangingPeriod":0,"PeriodChangedTest::testChangingPeriodByMethod":0,"PeriodIntervalTest::testChangingPeriodForMonth":0}}
```

Comment:
Add this file to `.gitignore`

Reply:
i forgot to delete this file before commit, now it should be fine